src/entities/auth/api.ts

```
@@ -77,4 +80,14 @@ export const authApi = {
   logout: async (): Promise<void> => {
     // TODO: Implement logout logic
   },
+
+  saveGoogleInfo: async (data: Record<string, any>): Promise<any> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)